pkg/centralsensor/caps_list.go

```diff
@@ -40,3 +40,8 @@ const (
 	// DelegatedRegistryCap identifies the capability for a secured cluster to interact directly with registries (ie: for scanning images in local registries).
 	DelegatedRegistryCap SensorCapability = "DelegatedRegistryCap"
 )
+
+const (
+	// PingCap identifies the capability for a Central to receive ping messages.
+	PingCap CentralCapability = "CentralPing"
```

Comment:
Curious if it's more accurate to say this capability defines Central's ability to receive pings or to send pongs